moped-editor/src/views/projects/signalProjectTable/RenderSignalLink.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Link from "@material-ui/core/Link";
+
+const RenderSignalLink = ({ signals }) => {
+  return (
+    <span>
+      {signals.map((signal, index) => (
+        <>
+          {signal.knack_id ? (
```

Comment:
👍

Reply:
Is this code WSOD prone? Is it safe to assume there will always be a knack_id? If not, let's use conditional chaining